dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -283,27 +293,33 @@ DomainParticipantImpl::delete_subscriber(
   SubscriberImpl* the_servant = dynamic_cast<SubscriberImpl*>(s);
 
   if (!the_servant) {
-    ACE_ERROR((LM_ERROR,
-      ACE_TEXT("(%P|%t) ERROR: ")
-      ACE_TEXT("DomainParticipantImpl::delete_subscriber, ")
-      ACE_TEXT("Failed to obtain SubscriberImpl.\n")));
+    if (DCPS_debug_level > 0) {
+      ACE_ERROR((LM_ERROR,
+        ACE_TEXT("(%P|%t) ERROR: ")
+        ACE_TEXT("DomainParticipantImpl::delete_subscriber, ")
+        ACE_TEXT("Failed to obtain SubscriberImpl.\n")));
+    }
     return DDS::RETCODE_ERROR;
   }
 
   if (!the_servant->is_clean()) {
-    ACE_ERROR((LM_ERROR,
-               ACE_TEXT("(%P|%t) ERROR: ")
-               ACE_TEXT("DomainParticipantImpl::delete_subscriber, ")
-               ACE_TEXT("The subscriber is not empty.\n")));
+    if (DCPS_debug_level > 0) {
+      ACE_ERROR((LM_ERROR,
+                ACE_TEXT("(%P|%t) ERROR: ")
+                ACE_TEXT("DomainParticipantImpl::delete_subscriber, ")
+                ACE_TEXT("The subscriber is not empty.\n")));
```

Comment:
```suggestion
                 ACE_TEXT("(%P|%t) ERROR: ")
                 ACE_TEXT("DomainParticipantImpl::delete_subscriber, ")
                 ACE_TEXT("The subscriber is not empty.\n")));
```

And I can see there are more after this.